### PR TITLE
Added workflow build, publish, update override file & create PR

### PR DESCRIPTION
**Pull request by lisa-sigsci**

### CODE REVIEW

File: .github/workflows/release-and-pr.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        body: |
+         Match release kubernetes/ingress-nginx v${{ env.NEW_TAG }} 
+        draft: false
+        prerelease: false
+
+    - name: Update sigsci-values.yaml & create PR
+      if: ${{ steps.tag.outputs.NO_CHANGE == 'false' }}
+      run: |
+        sed -ie "s/tag:.*/tag: ${{ env.NEW_TAG }}/g" sigsci-values.yaml
+        rm sigsci-values.yamle
```
Comment (leaf-fastly) [inline .github/workflows/release-and-pr.yml:59]: What's this yamle file?

File: .github/workflows/release-and-pr.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        git config --local user.email "github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        git commit -am "Automated PR - Match version"
```
Comment (leaf-fastly) [inline .github/workflows/release-and-pr.yml:62]: "Match upstream version" is a little more descriptive.

File: .github/workflows/release-and-pr.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+name: Compare ingress version -- Build & publish if new version
```
Comment (leaf-fastly) [inline .github/workflows/release-and-pr.yml:1]: Are we publishing, or creating a PR/candidate release?

**Review by leaf-fastly — CHANGES_REQUESTED** [review]
I have questions!

File: .github/workflows/release-and-pr.yml (hunk 1 above)
Comment (lisa-sigsci) [reply .github/workflows/release-and-pr.yml:59]: Ughh.. That was from a test I did to use a backup file. Removing

File: .github/workflows/release-and-pr.yml (hunk 2 above)
Comment (lisa-sigsci) [reply .github/workflows/release-and-pr.yml:62]: Updated

File: .github/workflows/release-and-pr.yml (hunk 3 above)
Comment (lisa-sigsci) [reply .github/workflows/release-and-pr.yml:1]: We are publishing the container on docker hub and the github release/tag. The last piece is creating a PR for the update to the sigsci-values.yaml override file.

Comment (lisa-sigsci) [reply .github/workflows/release-and-pr.yml:1]: We could automate the approval and merge of the PR but I though we wanted a little visibility into the process

Comment (leaf-fastly) [reply .github/workflows/release-and-pr.yml:1]: I agree, we do. I'm questioning the descriptiveness of this name for the action: "Compare ingress version -- Build & publish if new version"

**Review by leaf-fastly — APPROVED** [review]
rad! we could run this earlier in the day so people on the east coast or europe could also approve/merge the PR same day, instead of making just west-coasters do it ;-)

**Pull request merged**